src/content/posts/4-way-handshake.md/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "4-Way Handshake - Wifi Authentication"
+summary: "In this article, you will learn how authentitcation works for WPA/WPA2 network"

```

Comment:
how WPA/WPA2 authentication works or how authentication works in WPA/WPA2 based networks

Reply:
done